doc/how-to-source-packages.rst

```diff
@@ -0,0 +1,309 @@
+How to get package *source code*
+================================
+
+We can **"source"** or obtain the package *source code* and we can leave a
+**"source"** marker in the package description that shows where to find the
+*source code*. This marker is the ``source-repository`` field and is described
+in the :ref:`package author<pkg-authors>` section.
```

Comment:
As this paragraph gives an overview over the guide, a sentence about `source-repository-package` and a ref to _pkg-consumers would be good (as it was before)

Reply:
I've added back the ref for this.